pkg/types/types.go

```diff
@@ -34,6 +34,8 @@ const (
 
 	EngineFrontendBlockDev = "tgt-blockdev"
 	EngineFrontendISCSI    = "tgt-iscsi"
+
+	VolumeHeadName = "volume-head"
```

Comment:
The original commit doesn't have this change. I guess you did not use cherry-pick; otherwise, it should be a cherry-pick commit in the message.

Reply:
Actually I do use `cherry-pick` but this change `VolumeHeadName = "volume-head"` https://github.com/longhorn/longhorn-engine/commit/64a19ede4aa457863c07226eba77b1baadf193ef did not backport to v1.3.x and v1.2.x, then I added it manually.